vcr/credential/types.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * Nuts node
+ * Copyright (C) 2021 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package credential
+
+const (
+	// NutsOrganizationCredentialType is the VC type for a NutsOrganizationCredentialType
+	NutsOrganizationCredentialType = "NutsOrganizationCredential"
+	// DefaultCredentialType is the default credential type required for every credential
+	DefaultCredentialType = "VerifiableCredential"
```

Comment:
this const, and `DefaultContext` could be in `go-did`?

Reply:
It's becoming more annoying to use go-did in our project and go-did is becoming ever more opinionated....

Reply:
https://github.com/nuts-foundation/go-did/pull/25